port/win32/omrfiletext.c

```diff
@@ -96,50 +94,51 @@ omrfile_read_text(struct OMRPortLibrary *portLibrary, intptr_t fd, char *buf, in
 intptr_t
 omrfile_write_text(struct OMRPortLibrary *portLibrary, intptr_t fd, const char *buf, intptr_t nbytes)
 {
-	intptr_t result;
-	intptr_t i;
-	int newlines = 0, highchars = 0;
+	intptr_t result = 0;
+	intptr_t i = 0;
+	int newlines = 0;
+	int highchars = 0;
 	char stackBuf[512];
 	char *newBuf = stackBuf;
-	intptr_t newLen;
+	intptr_t newLen = 0;
 
 	/* scan the buffer for any characters which need to be converted */
 	for (i = 0; i < nbytes; i++) {
-		if (buf[i] == '\n') {
+		if ('\n' == buf[i]) {
 			newlines += 1;
-		} else if ((uint8_t)buf[i] & 0x80) {
+		} else if (0 != ((uint8_t)buf[i] & 0x80)) {
```

Comment:
```suggestion
		} else if (OMR_ARE_ANY_BITS_SET((uint8_t)buf[i], 0x80)) {
```

Reply:
Fixed in 901ae9e35bb330b5ce4fc1f5d0ea8f4b7ded11ae; also in unix version.